RedHat-UX/red-hat-design-system
# fix(footer): accessible logo in universal

## What I did

1. Fixed `<rh-global-footer>` slotted default logo so that link uses `aria-label` (instead of incorrectly using `alt` attribute) and its SVG uses a `<title>` element (instead of incorrectly using `title` attribute).
2. Added a controller that checks slotted links and buttons to ensure light DOM has readable link text and button labels. 

## Testing Instructions

1. Use Axe Dev Tools to check `<rh-footer>` and `<rh-global-footer>` (@hellogreg only).
2. Change the footer and global footer demo HTML locally to include links that have no text, and verify that the `lightdomAccessibleTextController` places logger warnings in the console.

## Notes to Reviewers

We may want to use this else where or consider something ab bit more comprehensive like https://github.com/RedHat-UX/red-hat-design-system/issues/813
